src/pages/App.tsx

```diff
@@ -231,10 +234,15 @@ export default function App() {
                       browserRouterEnabled && hash ? <Navigate to={hash.replace('#', '')} replace /> : <Landing />
                     }
                   />
-
-                  <Route path="tokens" element={<Tokens />}>
-                    <Route path=":chainName" />
-                  </Route>
+                  {infoExplorePageEnabled ? (
+                    <Route path="explore" element={<Explore />}>
+                      <Route path=":chainName" />
+                    </Route>
+                  ) : (
+                    <Route path="tokens" element={<Tokens />}>
+                      <Route path=":chainName" />
+                    </Route>
+                  )}
```

Comment:
I don't think we need all this duplicated code, I would suggest just routing to the previous component and possibly renaming it to `Explore`
```suggestion
                  <Route path={ infoExplorePageEnabled ? 'explore' : 'tokens'} element={<Tokens />}>
                      <Route path=":chainName" />
                    </Route>
```